tests/suites/test_suite_cipher.aes.data

```diff
@@ -2015,243 +2015,243 @@ depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CBC
 test_vec_crypt:MBEDTLS_CIPHER_AES_256_CBC:MBEDTLS_DECRYPT:"0000000000000000000000000000000000000000000000000000000000000000":"00000000000000000000000000000000":"5c9d844ed46f9885085e5d6a4f94c7d7":"014730f80ac625fe84f026c60bfd547d":0:0
 
 AES-128-CBC crypt Encrypt NIST KAT #1 PSA
-depends_on:MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CBC
+depends_on:MBEDTLS_USE_PSA_CRYPTO:!MBEDTLS_DEPRECATED_REMOVED:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CBC
```

Comment:
I think the extra dependency should be `MBEDTLS_TEST_DEPRECATED` (see comment in the test function).